features/cellular/framework/targets/QUECTEL/UG96/QUECTEL_UG96_CellularPower.cpp

```diff
@@ -23,7 +23,9 @@ using namespace mbed;
 
 QUECTEL_UG96_CellularPower::QUECTEL_UG96_CellularPower(ATHandler &atHandler) : AT_CellularPower(atHandler)
 {
-
+#if MODEM_ON_BOARD
+	::onboard_modem_init();
```

Comment:
CellularPower API is to be removed in feature-cellular-refactor, and power functionality need to be moved to `targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_WIO_3G/ONBOARD_<modem>`.